experiment/autobumper/bumper/bumper.go

```diff
@@ -555,24 +527,38 @@ func HasChanges() (bool, error) {
 	return len(strings.TrimSuffix(string(combinedOutput), "\n")) > 0, nil
 }
 
-func makeCommitSummary(images map[string]string) (string, error) {
-	version, err := getNewProwVersion(images)
-	if err != nil {
-		return "", err
+func getPrefixesString(prefixes []Prefix) string {
```

Comment:
worth it to reuse `getAllPrefixes`? 
`strings.Join(getAllPrefixes(prefixes), ", ")`

Reply:
Good catch!